python/src/llvm.cc

```diff
@@ -302,6 +303,22 @@ void init_triton_llvm(py::module &&m) {
         PassBuilder pb(nullptr /*targetMachine*/, tuningOptions, std::nullopt,
                        instrCbPtr);
 
+        std::string pluginFile =
+            mlir::triton::tools::getStrEnv("LLVM_PASS_PLUGIN");
+
+        if (!pluginFile.empty()) {
+          // TODO: Add some logging here that we inserted a pass into the LLVM
+          // pass pipeline
+          auto passPlugin = llvm::PassPlugin::Load(pluginFile);
+          if (!passPlugin) {
+            llvm::Error Err = passPlugin.takeError();
+            std::string ErrMsg =
+                "Pass Plugin Error: " + llvm::toString(std::move(Err));
```

Comment:
Any reason that we don't also include the `pluginFile` as part of the error message? For errors we should be as verbse as possible--that gives more context to the developer about the problem. It could save quite some debugging and head scratching with a bit more information if the problem for example comes from specifying the wrong pass plugin or whatever.

Reply:
The pluginFile will be include in the runtime exception from the llvm::Error ErrMsg. So you'll, as is, get something like:

RuntimeError: Pass Plugin Error: Could not load library '/home/libPassPlugin.so': /home/libPassPlugin.so: cannot open shared object file: No such file or directory

Reply:
Cool sounds good then!